src/README.md

```diff
@@ -8,7 +8,7 @@ React Cookie banner dismissable with just a scroll!
 | **children** | <code>union(ReactChildren &#124; Function)</code> |  | *optional*. Custom component rendered if user has not accepted cookies |
 | **message** | <code>String</code> |  | *optional*. Message written inside default cookie banner |
 | **onAccept** | <code>Function</code> | <code>"onAccept"</code> | *optional*. Called when user accepts cookies |
-| **link** | <code>{msg: ?String, url: String, target: ?"_blank" &#124; "_self" &#124; "_parent" &#124; "_top" &#124; "framename"}</code> |  | *optional*. Object with infos used to render a link to your cookie-policy page |
+| **link** | ```<a href="http://demo.url" target="_blank">Link text</a>``` |  | *optional*. JSX element to link to your cookie-policy page |
```

Comment:
Since this is the type of the prop, I would write `JSX.Element` here.

Reply:
I'm not sure which JSX element would be best - React on it's own doesn't have any `link` component. And using `<Link>` from `react-router` (or any other) would suggest that `react-cookie-banner` works only with this one routing library.

Reply:
Sorry, maybe I wasn't clear in my comment.

Since this is the *type* of the element, there's no need to pick a specific one. Just writing exactly `JSX.Element` is fine :)